### PR TITLE
static app with noddy content

### PR DESCRIPTION
**Pull request by chilliwilly77**
Convert hell-world in the lmt-app-moved app:
 * remove db 
 * change port for docker file to be 80
 * rename app
 * functionality to serve a single specific static file based on express_base_path

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -71,13 +76,14 @@ app.use(assetPath, cacheHeaders);
...
   'vendor', 'govuk_template_mustache_inheritance', 'assets')));
 
+app.use(`${assetPath}images`, express.static(path.join(__dirname, 'assets', 'images')));
+
 app.use(assetPath, express.static(path.join(__dirname, '..', 'dist', 'public')));
 
 app.use(helmet.noCache());
 
 app.use(`${basePath}/`, indexController);
 app.use(`${basePath}/`, cookieController);
```
Comment (will518) [inline app/app.js]: Do we still need the cookie controller?

File: gulpfile.js [hunk 2]
```diff
@@ -42,19 +45,23 @@ gulp.task('browserify', () =>
...
     'node_modules/jquery/dist/jquery.min.js',
-  ]).pipe(gulp.dest('dist/public/js'))
+  ]).pipe(gulp.dest(`${stagePath}js`))
 );
 
 gulp.task('js', ['browserify', 'js-vendor']);
 
-gulp.task('css', () =>
+gulp.task('fonts', () =>
+  gulp.src('node_modules/font-awesome/fonts/*').pipe(gulp.dest(`${stagePath}fonts`))
```
Comment (will518) [inline gulpfile.js]: Should we move the `.pipe` onto a new line?

File: gulpfile.js [hunk 3]
```diff
@@ -63,26 +70,35 @@ gulp.task('css', () =>
...
-    'dist/public/**/*.css',
-    'dist/public/**/*.js',
-    'dist/public/**/*.{jpg,png,jpeg,gif,svg}'])
+    `${stagePath}**/*.html`,
+    `${stagePath}**/*.css`,
+    `${stagePath}**/*.js`,
+    `${stagePath}**/*.{jpg,png,jpeg,gif,svg,otf,eot,ttf,woff,woff2}`])
     .pipe(debug())
     .pipe(rev())
-    .pipe(revDelOriginal())
```
Comment (will518) [inline gulpfile.js]: How comes we're removing this? Is it no longer required?

Comment (chilliwilly77) [reply gulpfile.js]: I copied the changes made by @mangogrin from cya. When I reviewed that change in cya it seemed to work nicely

File: app/app.js (hunk 1 above)
Comment (chilliwilly77) [reply app/app.js]: yeah we need a cookie page on all the apps, govuk requirement

**Pull request merged**